### DISCUSSION TITLE
Best way to help

### DISCUSSION THREAD

**Discussion by inventor321** [iSoron/uhabits#1276] [General Discussion] [answered]
What would be the best way to help out the project for a person that knows a little bit of coding and for a person that doesn't know any coding. (I would imagine trying out the new beta versions of the app could help.)

**Comment by hiqua** [comment, marked as answer]
It's mostly there: https://github.com/iSoron/uhabits#contributing

I don't know what the current beta is and how often it's updated though.

Thanks for helping out!

**Reply by hiqua** [reply]
Ah and feel free to ask for more here or in the chat room: https://gitter.im/loophabits/dev
